### PR TITLE
Temporarily add new fields

### PR DESCRIPTION
**Pull request by barry800414**
[sector]
[has_overtime_salary]
[is_overtime_salary_legal]
[has_compensatory_dayoff]

interface: 
image

Please merge after backend is ready

### CODE REVIEW

**Comment by barry800414** [comment]
@mark86092  I need help, for backend server

**Comment by barry800414** [comment]
should merge to master, close it

**Comment by mark86092** [comment]
github can set mege to target, reopen

File: js/index.js [hunk 1]
```diff
@@ -299,7 +320,11 @@ function submitForm() {
             day_real_work_time: $("#day_real_work_time").val(),
             overtime_frequency: $("#overtime_frequency_input input[name='frequency']:checked").val(),
             email: $("#email").val(),
-        },
+            sector: $("#sector").val(),
+            has_overtime_salary:$("#has_overtime_salary_input input[name='has_overtime_salary']:checked").val(),
+            is_overtime_salary_legal: $("#is_overtime_salary_legal_input input[name='is_overtime_salary_legal']:checked").val(),
+            has_compensatory_dayoff:$("#has_compensatory_dayoff_input input[name='has_compensatory_dayoff']:checked").val(),
```
Comment (mark86092) [inline js/index.js:329]: $("#has_compensatory_dayoff_input input[name='has_compensatory_dayoff']:checked")

如果沒有一個 input 被 checked 會得到什麼？

**Comment by mark86092** [comment]
I reviewed the logic part

File: js/index.js (hunk 1 above)
Comment (barry800414) [reply js/index.js:329]: 我之前用chrome測，如果沒有任何一個被選，$("#has_compensatory_dayoff_input input[name='has_compensatory_dayoff']:checked")加上.val()會回傳undefined。
剛剛chrome, firefox, IE測試都是一樣的結果

仔細查一下網路的說明，看起來即使找不到，jquery還是會回傳一個jquery object，永遠不會回傳null。
http://stackoverflow.com/questions/1302428/what-does-jquery-actually-return

**Pull request merged**